internal/users/tests/tests.go

```diff
@@ -95,6 +104,33 @@ func Mockgpasswd(_ *testing.T) {
 
 	if _, err := f.Write([]byte(strings.Join(args, " ") + "\n")); err != nil {
 		fmt.Fprintf(os.Stderr, "Mock: error while writing in file: %v", err)
+		f.Close()
 		os.Exit(1)
 	}
 }
+
+// Mockgetusers is the getent mock.
+func Mockgetusers(_ *testing.T) {
+	if os.Getenv("GO_WANT_HELPER_PROCESS") == "" {
+		return
+	}
+
+	args := os.Args
+	for len(args) > 0 {
+		if args[0] != "--" {
+			args = args[1:]
+			continue
+		}
+		args = args[1:]
+		break
+	}
+
+	if args[0] == "nss-fail" {
```

Comment:
See, here, you don’t use slices.Contains for instance :)

Reply:
Well, I could use it here as well, but there was no need for it due to the way the `MockGetusers` is implemented (we specify the return that we want from the mock directly as arguments).

See my reply on the other comment on why I changed to `slices.Contains()`